lib/hooks/useUpdateHeadTitle.tsx

```diff
@@ -0,0 +1,13 @@
+import { useEffect } from "react";
+
```

Comment:
@timarney do you think `<title>mytitle</title>` could work? I tried this and ran into a variety of issues

Reply:
Tried and well and no doesn't work --- the title gets over-written